play-scalatest/src/test/scala/play/grpc/scalatest/PlayScalaTestSpec.scala

```diff
@@ -26,6 +27,9 @@ class PlayScalaTestSpec
     with ScalaFutures
     with IntegrationPatience {
 
+  // TODO: This override is not working
+  override def testServerFactory = new SslTestServerFactory
```

Comment:
@mkurz I tried to override the `DefaultTestServerFactory` to enable SSL but the error persists. This factory should be used in `org.scalatestplus.play.BaseOneServerPerTest#newServerForTest`.

Reply:
I am aware of that, I made it working yesterday locally already. Pushing soon.

Reply:
It's not working because we override `newServerForTest` in `ServerGrpcClient`. When we remove that override it works. Also the methods to override changed in https://github.com/playframework/scalatestplus-play/pull/410